test/test_transforms.py

```diff
@@ -194,61 +192,55 @@ def test_mfcc(self):
         torch_mfcc_norm_none = mfcc_transform_norm_none(audio_scaled)
 
         norm_check = torch_mfcc.clone()
-        norm_check[:, :, 0] *= np.sqrt(n_mels) * 2
-        norm_check[:, :, 1:] *= np.sqrt(n_mels / 2) * 2
+        norm_check[:, :, 0] *= math.sqrt(n_mels) * 2
+        norm_check[:, :, 1:] *= math.sqrt(n_mels / 2) * 2
 
         self.assertTrue(torch_mfcc_norm_none.allclose(norm_check))
 
-    def test_librosa_consistency(self):
+    def _test_librosa_consistency_helper(self, n_fft, hop_length, power, n_mels, n_mfcc, sample_rate):
         try:
             import librosa
             import scipy
         except ImportError:
+            self.skipTest("Ensure that librosa and scipy are installed")
```

Comment:
This should be used as an annotation "@unittest.skip("[...]")" on the method that calls this. Also, issue a warning instead of an error. "Librosa and scipy are not available" or such.